Modified title, hyphens, guidance, unordered list (#547)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/formatting.adoc

For commands and command outputs in code blocks, observe the correct markup for user-replaced values, as described in xref:user-replaced-values[] and xref:user-replaced-values-xml[].

[[explain-commands-variables-in-code-blocks]]

> == Explanation of commands and variables used in code blocks

Reviewer: I don't actually like it under that section, because now it's hidden from the TOC. But I'd like to hear @IngridT1 weigh in on what she thinks is the best placement

> == Explanation of commands and variables used in code blocks

Reviewer 2: TBH, I like where the information was located originally--directly after "Commands in code blocks". It makes the most sense to have the "Explanation of commands and variables in code blocks" directly after our first mention of commands in code blocks. 

I had never noticed that this list was alphabetical. Maybe it would make more sense to change the order of the items to be grouped by topic.

> == Explanation of commands and variables used in code blocks

Reviewer 2: With all the commentary, I'm finding it confusing to review the PR. Is there a way to look at just the latest version of the PR before we merge it?

> == Explanation of commands and variables used in code blocks

Reviewer: @IngridT1 are you familiar with how to build PRs locally to review? If not, it's pretty straightforward and I'd be happy to show you. It really does make a world of difference to be able to see how it renders (plus be able to test xrefs, etc.) before merging.

> == Explanation of commands and variables used in code blocks

Reviewer 2: @bergerhoffer BrianB showed me how to do that! I guess that I didn't realize that would help with my confusion. Let me try it. I'll ping you if I'm still having issues.

> == Explanation of commands and variables used in code blocks

Reviewer 2: I did a local build, @bergerhoffer and @Preeticp. I much prefer having this information in the original location, even though those headings are no longer alphabetical. It helps a lot to be able to see the section in the ToC. It also makes the most sense to have these 2 sections together.

For commands and command outputs in code blocks, observe the correct markup for user-replaced values, as described in xref:user-replaced-values[] and xref:user-replaced-values-xml[].

[[explain-commands-variables-in-code-blocks]]
=== Explanation of commands and variables used in code blocks

> To explain commands, lines of code, or variables (user-replaced values) in a code block, do not use callouts. DITA does not support callouts, hence there is no semantic markup for an AI agent to interpret or output them correctly.

Reviewer 2: I did not notice this when I originally reviewed this PR, but I would not use "hence" in this sentence. We also should avoid using parentheses. How about this instead?

```suggestion
To explain commands, lines of code, or user-replaced values in a code block, do not use callouts. DITA does not support callouts, and an AI agent cannot interpret or output them correctly.
```

IBM guidelines about parentheses: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses

Author: IMHO, we do not provide the reasoning for every guidance captured in the SSG. I am not really sure we need the reasoning mentioned here, it was provided in the mail sent out to CCS.

Author: The AI agent cannot interpret or output them correctly because DITA does not provide the semantic markup for it to interpret callouts. I used "hence" in this context, not sure using "and" is right in this context.

> Instead, follow the code block with the relevant explanation or description of the elements, using the following guidelines.

Author: Fixed

+
.Example AsciiDoc: A simple sentence explaining a single command

[source,terminal]
----
$ hcp create cluster <platform> --help 
----
+
Use the `hcp create cluster` command to create and manage hosted clusters. The supported platforms are `aws`, `agent`, and `kubevirt`.

* Use a definition list to explain multiple options, parameters, variables, placeholders, or UI elements.
**  List the parameters/variables in the order in which they appear in the code block.
**  Introduce definition lists with "where:" and begin each variable description with "Specifies".
+
.Example AsciiDoc: A definition list explaining user-replaced values

[source,yaml,subs="+attributes,+quotes"]
----
$ cat <<EOF | oc -n <my_product_namespace> create -f -
apiVersion: v1
kind: Secret
metadata:
 name: <my_product_database_certificates_secrets> 
type: Opaque
stringData:
 postgres-ca.pem: |-
  -----BEGIN CERTIFICATE-----
  <ca_certificate_key> 
 postgres-key.key: |-
  -----BEGIN CERTIFICATE-----
  <tls_private_key> 
 postgres-crt.pem: |-
  -----BEGIN CERTIFICATE-----
  <tls_certificate_key> 
  # ...
EOF
----
+

> where:

Reviewer 3: This needs to be a full sentence to comply with general guidance, accessibility, and localization requirements.

```suggestion
Use the following values in the configuration file:
```

Reviewer 2: @jafiala Because IBM Style recommends "where", I think we should keep this guidance as is. I understand your concerns about accessibility and localization. Please open a new [issue](https://github.com/redhat-documentation/supplementary-style-guide/issues), and we can discuss the pros and cons there. 

@Preeticp It's my understanding that these examples are guidance and that we can choose to deal with the description list differently. Is that correct? Would it be okay if Jan's team decides to use a complete sentence instead of "where"? Like this:

`Use the following values in the command:`

      - build-image
...
----
+

> ** `workspaces` specifies the list of pipeline workspaces shared between the tasks defined in the pipeline. A pipeline can define as many workspaces as required. In this example, only one workspace named `shared-workspace` is declared.

Reviewer: I agree on lowercase here - if there is a colon or something in between, I think it would be capitalized (like how the DL option is). But since it's a continuation (like `<workspaces>` being the first "word" of the sentence), I think it makes sense to be lowercase to read as a sentence.
